sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -524,6 +524,8 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * <li>`comment` (default empty string): sets a single character used for skipping lines
    * beginning with this character. By default, it is disabled.</li>
    * <li>`header` (default `false`): uses the first line as names of columns.</li>
+   * <li>`enforceSchema` (default `true`): Forcibly apply the specified or inferred schema to
+   * datasource files. If it is set, CSV headers are ignored. </li>
```

Comment:
Also need to explain the behavior when it is set to false.

Reply:
Also need to explain this conf does not have an effect when `header` is set to `false`